pkg/applier/applier.go

```diff
@@ -600,7 +598,7 @@ func (s *supervisor) applyInner(ctx context.Context, eventHandler func(Event), d
 			klog.Info(e.ErrorEvent)
 			err := e.ErrorEvent.Err
 			if util.IsRequestTooLargeError(err) {
-				err = largeResourceGroupError(err, idFromInventory(s.inventory))
+				err = largeResourceGroupError(err, idFromInventoryInfo(s.invInfo))
```

Comment:
nit: now that Info has its own ID, it might make sense to rename idFromInventoryInfo to coreIDFromInventoryInfo just to disambiguate.